src/network/networkd-network.h

```diff
@@ -157,6 +157,13 @@ struct Network {
         AddressFamilyBoolean link_local;
         bool ipv4ll_route;
 
+        /* IPv6 prefix delegation support */
+        bool router_prefix_delegation;
+        uint32_t router_lifetime;
```

Comment:
Yes, usec_t router_prefix_delegation_usec so that it is named consistently with dhcp_server_default_lease_time_usec.